Keas.Mvc/ClientApp/components/Keys/CreateKey.tsx

```diff
@@ -155,10 +157,13 @@ export default class CreateKey extends React.Component<IProps, IState> {
             valid = false;
         } else if (!key.code) {
             valid = false;
-            error = "You must give this key a name.";
+            error = "You must give this key a code.";
         } else if (key.code.length > 64) {
             valid = false;
-            error = "The name you have chosen is too long";
+            error = "The code you have chosen is too long";
+        } else if (this.props.keyCodes.indexOf(key.code) !== -1) {
```

Comment:
i'm thinking this will do what you want to make sure they aren't already using that key code actively.  An extra DB check would be nice but i don't think it's that necessary as the downside is small